### ISSUE TITLE
Hover table hides info

### ISSUE THREAD

**Issue by richardwerkman** [stryker-mutator/mutation-testing-elements#2144]
When hovering the table all usefull numbers dissapear:

![image](https://user-images.githubusercontent.com/14224500/195847356-38579243-c6a9-4255-afb5-ec899faf509d.png)
![image](https://user-images.githubusercontent.com/14224500/195847418-96738bc3-48a5-4121-9304-ed8acacf0dad.png)

I would want to see that all numbers are still visible when hovering

**Comment by nicojs** [comment]
Yeah pretty sure. I think you mean older version. I.e. less broken version
